src/igvfd/schemas/measurement_set.json

```diff
@@ -163,6 +163,21 @@
                 "igvf_utils": "john-doe:measurement_set01, TSTDS98084730"
             }
         },
+        "control_type": {
+            "title": "Control Type",
+            "description": "This property describes the type of control this file-set represents.",
+            "comment": "This property is overwritten by the subclasses to define specific enum values.",
```

Comment:
Since Measurement Set is the subclass, I don't think it is necessary to include the comment again